demisto_sdk/common/hook_validations/integration.py

```diff
@@ -438,7 +439,26 @@ def is_docker_image_changed(self):
     def is_id_equals_name(self):
         """Check whether the integration's ID is equal to its name
 
-            Returns:
-                bool. Whether the integration's id equals to its name
-            """
+        Returns:
+            bool. Whether the integration's id equals to its name
+        """
         return super(IntegrationValidator, self)._is_id_equals_name('integration')
+
+    def is_not_valid_display_configuration(self):
+        """Validate that the display settings are not empty for non-hidden fields.
+
+        Returns:
+            bool. Whether the dispaly is there for non-hidden fields.
+        """
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            hidden_value = configuration_param['hidden']
+            configuration_display = configuration_param['display']
+
+            if not hidden_value:
+                if not configuration_display:
```

Comment:
should be changed to one statement if we're not adding an elif/else to one of them